src/main/java/com/greenfoxacademy/responses/ListedItemsResponse.java

```diff
@@ -0,0 +1,21 @@
+package com.greenfoxacademy.responses;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+/**
+ * Created by BSoptei on 2/1/2017.
+ */
+public class ListedItemsResponse extends CustomServerResponse {
+    private String body;
```

Comment:
currently it is converted into a String from a List<Contact> (this is one type of a solution for the do-not-display-all-user-data problem)